packages/core/admin/admin/src/content-manager/pages/EditView/components/DocumentActions.tsx

```diff
@@ -274,7 +276,7 @@ const DocumentActionsMenu = ({
             })}
         </VisuallyHidden>
       </Menu.Trigger>
-      <Menu.Content top="4px" popoverPlacement="bottom-end">
+      <Menu.Content top="4px" maxHeight={undefined} popoverPlacement="bottom-end">
```

Comment:
Just wondering why you need to pass maxHeight as undefined?

Reply:
because by default the `Menu.Content` has a fixed height to let menu items scroll inside them, I kind of think it's a bit dumb and i'm wondering if from V2 the default isn't set and you set it to w/e.... thoughts?